metadata-editor/app/model/UsageRightsProperty.scala

```diff
@@ -72,4 +79,12 @@ object UsageRightsProperty {
 
     case _ => List()
   }
+
+  private def illustrationProperties(u: UsageRights) = u match {
+    case _:ContractIllustrator     => List(
+      UsageRightsProperty("creator", "Illustrator", "string", true, Some(sortList(commissionedIllustrators))))
```

Comment:
Your confusion is my stupidity, we would only have a list of contract illutrators.